bot/kodiak/test_queries.py

```diff
@@ -668,8 +674,8 @@ def test_get_commit_authors_error_handling() -> None:
     pull_request_data = {
         "commitHistory": {
             "nodes": [
-                {"commit": {"author": {"user": {}}}},
-                {"commit": {"author": {}}},
+                {"commit": {"author": {"user": None}}},
+                {"commit": {"author": None}},
```

Comment:
This didn't parse and I don't think the GitHub API will return responses like this.